src/components/Settings/Settings.js

```diff
@@ -1,165 +1,30 @@
+import { useNavigation } from "@react-navigation/native";
 import { useQueryClient } from "@tanstack/react-query";
 import { updateUsers } from "api/users";
-import { ActivityIndicator, ViewWrapper } from "components/SharedComponents";
-import { t } from "i18next";
-import type { Node } from "react";
+import {
+  ActivityIndicator,
+  Body2, Button, Heading4, RadioButtonRow,
+  ViewWrapper
+} from "components/SharedComponents";
 import React, { useEffect, useState } from "react";
 import {
-  Button,
-  Pressable,
   SafeAreaView,
-  ScrollView,
-  StatusBar,
-  Text,
-  View
+  StatusBar, View
 } from "react-native";
+import { useTranslation } from "sharedHooks";
 import useAuthenticatedMutation from "sharedHooks/useAuthenticatedMutation";
 import useUserMe from "sharedHooks/useUserMe";
-import { textStyles, viewStyles } from "styles/settings/settings";
-
-import SettingsAccount from "./SettingsAccount";
```

Comment:
Should we also remove all those files if they are not used anymore?

Reply:
We probably don't need the styles, but I'm going to leave the bulk of the existing Settings components in case we change our minds and want to ditch the webview approach.